test/fuzzing/find_fuzz.cpp

```diff
@@ -37,6 +38,25 @@ struct QueryFixture {
     wq_ = std::make_shared<iroha::ametsuchi::MockWsvQuery>();
     EXPECT_CALL(*storage_, getBlockQuery()).WillRepeatedly(Return(bq_));
     EXPECT_CALL(*storage_, getWsvQuery()).WillRepeatedly(Return(wq_));
+    EXPECT_CALL(*storage_, createQueryExecutor(_, _))
+        .WillRepeatedly([&](const auto & /* pending_txs */,
+                            const auto &response_factory) {
+          using namespace ::testing;
+          auto query_executor =
+              std::make_unique<iroha::ametsuchi::MockQueryExecutor>();
+          EXPECT_CALL(*query_executor, validateAndExecute_(_))
+              .WillRepeatedly(
+                  Return(response_factory
+                             ->createErrorQueryResponse(
+                                 shared_model::interface::QueryResponseFactory::
+                                     ErrorQueryType::kStatefulFailed,
+                                 "I'm a teapot",
+                                 418,
+                                 shared_model::crypto::Hash{"BADD00DE"})
+                             .release()))
+              .Times(AtMost(1));
```

Comment:
```suggestion
              .WillOnce(
                  Return(response_factory
                             ->createErrorQueryResponse(
                                 shared_model::interface::QueryResponseFactory::
                                     ErrorQueryType::kStatefulFailed,
                                 "I'm a teapot",
                                 418,
                                 shared_model::crypto::Hash{"BADD00DE"})
                             .release()));
```

Reply:
i didn't want to limit the minimum amount of calls since it is not directly related to the test, but your suggestion may be still reasonable, because if some day we will avoid this being called, the expectation can be removed.